src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -4733,6 +4753,188 @@ public override (int e, int f) M((int g, int h) y)
         }
 
         [Fact]
+        public void OverrideTupleMethodWithDifferentNamesInParameterType()
```

Comment:
Didn't see any tests that verify we can override with `ValueTuple<int, int>`, just the tuple syntax form `(int, int)`.